frontend/packages/kubevirt-plugin/src/components/modals/disk-modal/disk-modal.tsx

```diff
@@ -142,33 +142,27 @@ export const DiskModal = withHandlePromise((props: DiskModalProps) => {
   let resultVolume;
   if (source.requiresVolume()) {
     // update just Disk for unknown sources
-    resultVolume = VolumeWrapper.initializeFromSimpleData(
-      {
-        name,
-        type: source.getVolumeType(),
-        typeData: {
-          name: resultDataVolumeName,
-          claimName: pvcName,
-          image: containerImage,
-        },
+    resultVolume = VolumeWrapper.initializeFromSimpleData({
```

Comment:
Personally, I think the word "simple" is too ambiguous here, maybe we can rename it (later) to better convey the purpose.

Reply:
I am for changing that if we come up with a better name